backend/pom.xml

```diff
@@ -17,6 +17,7 @@
         <java.version>19</java.version>
         <sonar.organization>fullofpit</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+        <sonar.exclusions>**/config/**</sonar.exclusions>
```

Comment:
Not described in the PR changelist

Reply:
Oh damn, sorry! Won't happen again

Reply:
Well need to add this to the changelist above. You can still edit your PR